crates/ide/src/references.rs

```diff
@@ -319,12 +319,19 @@ fn func() {
 fn test() {
     test_func();
 }
+
+#[tokio::test]
```

Comment:
`#[tokio::test]` is already mark as test, cause it will be expand to `#[::core::prelude::v1::test]`

Reply:
I thought it's not working because I saw a lot of references in `tokio::test`, but the problem with detection seems more general. 

e.g. a `vec![]` is enough to break the current filter

```rust
struct Point {
    x: f64,
    y: f64,
}

#[cfg(test)]
mod tests {
    use super::*;

    #[test]
    fn it_works() {
        assert_eq!(2 + 2, 4);
        let vec = vec![Point { x: 1.0, y: 2.0 }];
        assert_eq!(vec.len(), 1);
    }
}

```